frontend/components/FieldImage/FieldImageEdit.jsx

```diff
@@ -174,26 +180,40 @@ export default class FieldImageEdit extends Component {
     }
   }
 
-  handleFileChange(file) {
+  handleFileChange(files) {
     const {
       config,
       onChange,
       schema
     } = this.props
-    const reader = new FileReader()
-    
-    reader.onload = event => {
-      if (typeof onChange === 'function') {
-        onChange.call(this, schema._id, {
+    const singleFile = schema.settings && schema.settings.limit === 1
+
+    let processedFiles = []
+
+    for (let index = 0; index < files.length; index++) {
```

Comment:
`files` is an instance of [FileList](https://developer.mozilla.org/en/docs/Web/API/FileList), not Array, so it doesn't have a `forEach` method.